src/tests/performance/mod.rs

```diff
@@ -88,3 +86,16 @@ pub fn table_float_display(x: &f64) -> String {
 pub fn duration_as_ms(duration: Duration) -> f64 {
     duration.as_millis() as f64
 }
+
+/// Formats a [Table] with our style:
+///  - [pseudo style](Style) (todo - fix this link)
```

Comment:
I want this to link to [Style::pseudo()](https://docs.rs/tabled/0.2.1/tabled/style/struct.Style.html#method.pseudo), but I don't know how to link to an external crate's function?

Reply:
Should be just [pseudo style](https://docs.rs/tabled/0.2.1/tabled/style/struct.Style.html#method.pseudo) no?

Reply:
That works; but I was hoping for something a bit more automated? (i.e. link to the version we are actually running). What bothers me is that `(Style)` works just fine, but I can't get any further.

Reply:
Oh you mean the method? What about `[...](Struct::pseudo)`?